target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -23,4 +23,8 @@ case "$board" in
 		[ "$PHYNBR" = "0" ] && macaddr_add $(get_mac_label) 2 > /sys${DEVPATH}/macaddress
 		[ "$PHYNBR" = "1" ] && macaddr_add $(get_mac_label) 3 > /sys${DEVPATH}/macaddress
 		;;
+	tplink,eap660hd-v1)
+		base_mac=$(get_mac_binary /tmp/factory_data/default-mac 0)
+		macaddr_add $base_mac $(expr "$PHYNBR" + 1) > /sys${DEVPATH}/macaddress
+		;;
```

Comment:
One option is to use the recent ath11k MAC patching support to patch the MAC to caldata instead of setting it later.

Reply:
After further review of the stock image, the ethernet NIC and the 2.4GHz radio share the same MAC. I made the above change with that in mind. Is that okay, or should there be a different address for each NIC?

Reply:
MAC-s should reflect those in the stock FW, so if stock FW reuses the same then fine.